### ISSUE TITLE
ungit doesn't open

### ISSUE THREAD

**Issue by bax-** [codingtwinky/atom-ungit#50]
Yesterday Ungut stopped working – I don't know why.

When I toggle Ungut, nothing happens.

This is the output in the console:

```
atom-ungit.coffee [sm]:121 Users/bastian/.atom/packages/atom-ungit/node_modules/ungit/bin/ungit [-v] [-b]
[--cliconfigonly] [--gitVersionCheckOverride]

Options:
  -o, --gitVersionCheckOverride   Ignore git version check and allow ungit to
                                  run with possibly lower versions of git
                                                                [default: false]
  --ungitVersionCheckOverride     Ignore check for older version of ungit
                                                                [default: false]
  -b, --launchBrowser             Launch a browser window with ungit when the
                                  ungit server is started. --no-b or
                                  --no-launchBrowser disables this
                                                                 [default: true]
  --cliconfigonly                 Ignore the default configuration points and
                                  only use parameters sent on the command line
                                                                       [boolean]
  --port                          The port ungit is exposed on   [default: 8448]
  --urlBase                       The base URL ungit will be accessible from
                                                   [default: "http://localhost"]
  --rootPath                      The root path ungit will be accessible from
                                                                   [default: ""]
  --logDirectory                  Directory to output log files  [default: null]
  --logRESTRequests               Write REST requests to the log [default: true]
  --logGitCommands                Write git commands issued to the log
                                                                [default: false]
  --logGitOutput                  Write the result of git commands issued to the
                                  log                           [default: false]
  --bugtracking                   This will automatically send anonymous bug
                                  reports                        [default: true]
  --sendUsageStatistics           Google analytics for usage statistics
                                                                 [default: true]
  --authentication                True to enable authentication. Users are
                                  defined in the users configuration property
                                                                [default: false]
  --users                         Map of username/passwords which are granted
                                  access
  --showRebaseAndMergeOnlyOnRefs  Set to false to show rebase and merge on drag
                                  and drop on all nodes          [default: true]
  --maxConcurrentGitOperations    Maximum number of concurrent git operations
                                                                    [default: 4]
  --forcedLaunchPath              Define path to be used on open. Can be set to
                                  null to force the home screen
  --autoShutdownTimeout           Closes the server after x ms of inactivity.
                                  Mainly used by the clicktesting
  --maxNAutoRestartOnCrash        Maximum number of automatic restarts after a
                                  crash. Undefined == unlimited     [default: 0]
  --noFFMerge                     Don't fast forward git mergers. See git merge
                                  --no-ff documentation          [default: true]
  --autoFetch                     Automatically fetch from remote when entering
                                  a repository using ungit       [default: true]
  --dev                           Automatically does stash -> operation -> stash
                                  pop when you checkout, reset or cherry pick
                                                                 [default: true]
  --logLevel                      The logging level, possible values are none,
                                  error, warn, info, verbose, debug, and silly.
                                                               [default: "warn"]
  --launchCommand                 Specify a custom command to launch. `%U` will
                                  be replaced with the URL that corresponds with
                                  the working git directory.
  --allowCheckoutNodes            Allow checking out nodes (which results in a
                                  detached head)                [default: false]
  --allowedIPs                    An array of ip addresses that can connect to
                                  ungit. All others are denied   [default: null]
  --autoPruneOnFetch              Automatically remove remote tracking branches
                                  that have been removed on the server when
                                  fetching. (git fetch -p)       [default: true]
  --pluginDirectory               Directory to look for plugins
                                      [default: "/Users/bastian/.ungit/plugins"]
  --pluginConfigs                 No supported as a command line argument, use
                                  ungitrc config file.  See README.md
                                                                   [default: {}]
  --autoStashAndPop               Used for development purposes  [default: true]
  --fileSeparator                 OS dependent file separator     [default: "/"]
  --disableDiscardWarning         disable warning popup at discard
                                                                [default: false]
  --disableDiscardMuteTime        duration of discard warning dialog mute time
                                  should it be muted           [default: 300000]
  --lockConflictRetryCount        Allowed number of retry for git "index.lock"
                                  conflict                          [default: 3]
  --autoCheckoutOnBranchCreate    Auto checkout the created branch on creation
                                                                [default: false]
  --alwaysLoadActiveBranch        Always load with active checkout branch
                                                                [default: false]
  --numberOfNodesPerLoad          number of nodes to load for each git.log call
                                                                   [default: 25]
  --mergeTool                     the git merge tool to use when resolving
                                  conflicts                     [default: false]
  -h, --help                      Show help                            [boolean]
  -v, --version                   Show version number                  [boolean]
  --socketId                                                        [default: 0]

Examples:
  Users/bastian/.atom/packages/atom-ungit/  Run Ungit on port 8888
  node_modules/ungit/bin/ungit --port=8888
  Users/bastian/.atom/packages/atom-ungit/  Turn off REST logging but turn on
  node_modules/ungit/bin/ungit              git command log
  --no-logRESTRequests --logGitCommands

(anonymous function) @ atom-ungit.coffee [sm]:121emitOne @ events.js:101emit @ events.js:188readableAddChunk @ _stream_readable.js:176Readable.push @ _stream_readable.js:134onread @ net.js:543
atom-ungit.coffee [sm]:121 Unknown argument: socketId
(anonymous function) @ atom-ungit.coffee [sm]:121emitOne @ events.js:101emit @ events.js:188readableAddChunk @ _stream_readable.js:176Readable.push @ _stream_readable.js:134onread @ net.js:543
```
```
Unknown argument: socketId
(anonymous function) @ atom-ungit.coffee [sm]:121emitOne @ events.js:101emit @ events.js:188readableAddChunk @ _stream_readable.js:176Readable.push @ _stream_readable.js:134onread @ net.js:543
```

**Comment by codingtwinky** [comment]
This was related with recent strict enforcement of config variables and should be now fixed in ungit@1.1.14.  you can fix this by either upgrading ungit within atom plugin folder or upgrading your global ungit.

more on this are here.  https://github.com/FredrikNoren/ungit/pull/899

**Comment by bax-** [comment]
thanks @codingtwinky !
I ran `sudo -H npm update -g ungit` within `~/.atom/packages/atom-ungit` and the problem disappeared.
